addOns/websocket/src/main/zapHomeFiles/scripts/templates/websocketpassive/Passive WebSocket Scan Template.py

```diff
@@ -0,0 +1,84 @@
+"""
+Passive scan rules should not send messages
+Right click the script in the Scripts tree and select "enable"  or "disable"
+"""
+
+OPCODE_CONTINUATION = 0x0
+
+OPCODE_TEXT = 0x1
+OPCODE_BINARY = 0x2
+
+OPCODE_CLOSE = 0x8
+OPCODE_PING = 0x9
+OPCODE_PONG = 0xA
+
+RISK_INFO 	= 0
```

Comment:
Same comments as for the JavaScript script.

Reply:
Fixed in 17ea0de3a